tests/issues/issue-2651.php

```diff
@@ -0,0 +1,51 @@
+<?php
+declare(strict_types = 1);
+
+return [
+    'issue-XXXX-A' => [
```

Comment:
Please could you update the keys here too?

Reply:
oopsie